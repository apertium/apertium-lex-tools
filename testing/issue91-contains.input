^ver<vblex>/ablah<n>/other<n>$
^ver<vblex>/blah<n>/other<n>$
^vera<vblex>/ablah<n>/other<n>$
^vera<vblex>/blah<n>/other<n>$
^aver<vblex>/ablah<n>/other<n>$
^aver<vblex>/blah<n>/other<n>$
^avera<vblex>/ablah<n>/other<n>$
^avera<vblex>/blah<n>/other<n>$

^ver<vblex>/blaha<n>/other<n>$
