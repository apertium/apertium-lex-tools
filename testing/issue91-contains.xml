<rules>
	<rule weight="2.0">
		<match contains="ver" tags="vblex">
			<select suffix="blah"/>
		</match>
	</rule>
</rules>
