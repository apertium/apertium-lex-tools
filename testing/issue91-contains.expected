^ver<vblex>/ablah<n>$
^ver<vblex>/blah<n>$
^vera<vblex>/ablah<n>$
^vera<vblex>/blah<n>$
^aver<vblex>/ablah<n>$
^aver<vblex>/blah<n>$
^avera<vblex>/ablah<n>$
^avera<vblex>/blah<n>$

^ver<vblex>/blaha<n>/other<n>$
